sdk/freestanding-execution-engine/README.md

```diff
@@ -0,0 +1,36 @@
+# Goal of the exercise
+
+The purpose of this exercise is to get familiarised with how to load files from the host and into the Veracruz freestanding execution engine. The freestanding execution engine inside of the Veracruz instance allows a user's program to be executed.
+
+IMPORTANT: This exercise assume some familiarity with reading **Rust syntax** and **Rust's package manager named Cargo**. Both these areas are well explained at this [link](https://doc.rust-lang.org/book/title-page.html).
+
+# Steps:
+1. Install [Rust](https://www.rust-lang.org/tools/install) and [wasmtime](https://wasmtime.dev/).
```

Comment:
Why is installing `wasmtime` necessary?  Building the freestanding execution engine should pull in the `wasmtime` dependency from `crates.io`.

Reply:
I think I wrote this because I was experimenting with `wasmtime` outside of the Veracruz project as well. I will remove this part of the step.